app.js

```diff
@@ -1,36 +1,98 @@
 // Task 1 - Notes for JS Evaluation Predictions
 
 // List 3 expressions that surprised you or gave unexpected results, why?
+//'engineering.indexOf("neer")' surprised me because I didn't know what to expect. But as I looked over stuff I realized what was happening. The value being passed is a way of specifying a character if the string contains multiple of the same character.
+//2.5 * 'fish' gave me an unexpected result as well. I wasn't sure what the result would be but once I saw it I understood why
+//'engineering'.indexOf('G') I wasn't expecting -1 for the output. I figured i'd get undefined or false but I understand that -1 means not found
 
 // List 2 exprsssions you felt comfortable interpreting at a glance.
-
+'engineering'.indexOf('r') > -1
+'science'.indexOf('x') === -1
 // List 1 expression you are still 'shaky' on, why?
-
+//None...I understand each function. What I would like to know though is how we would get the last index of the second 'g' in 'engineering' using  indexOf()
```

Comment:
If you do `'engineering'.indexOf('g')` and you get back 2, then you know that the first g is at index 2 so then you can do `'engineering'.indexOf('g', 3)` and you would get the index of the first g that is after the second index. You can use this number to say what index you you want to start looking at.